src/components/Nav.vue

```diff
@@ -33,22 +33,35 @@
     >
 
       <div class="navbar-end">
-        <div class="navbar-item">
+        <div class="navbar-item" v-if="!getLog">
           <div class="buttons">
-            <router-link v-if="!getLog" to="/" class="button is-light">
+            <router-link to="/" class="button is-light">
               Se connecter
             </router-link>
-            <router-link v-if="getLog" to="/profile" class="button is-light">
-              Profile
-            </router-link>
-            <router-link v-if="getLog && hasUnconfiguredBackend" to="/create-account" class="button is-light">
+          </div>
+        </div>
+
+        <div class="navbar-item has-dropdown is-hoverable" v-if="getLog">
+          <a class="navbar-link">
+            {{ userProfile ? userProfile.name : 'inconnu' }}
+          </a>
```

Comment:
For some reason, I don't get anything diplayed in the bar. I didn't dig further to be honest. I hope it is not coming from me, it is still possible. Anyway, your rebase is welcome and quick double check is welcome on this point.

Reply:
I see. It's because it depended on the big PR #53 for the userprofile fetching in the store.